src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -984,8 +956,7 @@
     (let [subject (get-csr-subject csr)]
       (sling/throw+
        {:kind :disallowed-extension
-        :msg (format (str "CSR '%s' contains subject alternative names (%s), which are disallowed. "
-                          "Use `puppet cert --allow-dns-alt-names sign %s` to sign this request.")
+        :msg (tru "CSR ''{0}'' contains subject alternative names ({1}), which are disallowed. Use `puppet cert --allow-dns-alt-names sign {2}` to sign this request."
```

Comment:
Also the CLI command `puppet cert --allow-dns-alt-names sign {2}`, could this be a single string that's interpolated? Since I don't think this will be translated. So that all the translator sees is 
`"CSR ''{0}'' contains subject alternative names ({1}), which are disallowed. Use {2} to sign this request."`
@LarissaLane Can you give some guidance on this one? As I type that - does it now need comments for context? Oy vey, am I making a mountain out of a mole hill?